docs/README.txt

```diff
@@ -1,18 +1,21 @@
-All of the SunPy documentation is contained in the doc/source folder and code
-comments. To generate the documentation you must have Sphinx (as well as
-Numpydoc and astropy-helpers) installed on your computer. Enter
+All of the SunPy documentation is contained in the `docs` folder and code
+docstrings/comments. To generate the documentation you must have Sphinx (as
+well as Numpydoc and astropy-helpers) installed on your computer. Enter
```

Comment:
The list of doc requirements is a little longer than this now. I think all you *need* is `sphinx-astropy` though?

Reply:
This is why I plan to remove this file in a different PR. This file serves no real use as all this should be in the doc pages.

Reply:
@Cadair I tested building the docs in a fresh dev environment and found that `Sphinx` and `sphinx-astropy` are the only requirements to build the docs. However, things like the sunpy sphinx theme are missing from the built docs, so rather than enumerate the required packages here, I reworded the text to point to ``requirements/docs.txt``. I also made this change in `docs/dev_guide/documentation.rst`. Let me know if this works for you!